pkg/gce-pd-csi-driver/controller.go

```diff
@@ -201,7 +201,22 @@ const (
 
 var (
 	validResourceApiVersions = map[string]bool{"v1": true, "alpha": true, "beta": true, "staging_v1": true, "staging_beta": true, "staging_alpha": true}
-
+	// Hyperdisk types that support dynamic IOPS changes
```

Comment:
Will iops/throughput support ever be different?

And will there be new disk SKUs that support changing iops/throughput? (If so consider a command line argument so we can set this without a binary release of the PDCSI driver).

Reply:
Done